integration-tests-support/azure/src/main/java/org/apache/camel/quarkus/test/support/azure/AzureServiceBusTestResource.java

```diff
@@ -0,0 +1,118 @@
+    private static final Logger LOGGER = LoggerFactory.getLogger(AzureServiceBusTestResource.class);
+    private Map<String, String> initArgs = new LinkedHashMap<>();
+    private ComposeContainer container;
+
+    @Override
+    public void init(Map<String, String> initArgs) {
+        this.initArgs = initArgs;
+    }
+
+    @Override
+    public Map<String, String> start() {
+        final SmallRyeConfig config = ConfigUtils.configBuilder(true, LaunchMode.NORMAL).build();
+
+        final boolean realCredentialsProvided = System.getenv("AZURE_SERVICEBUS_CONNECTION_STRING") != null
+                && System.getenv("AZURE_SERVICEBUS_QUEUE_NAME") != null;
+
+        final boolean startMockBackend = MockBackendUtils.startMockBackend(false);
+        final Map<String, String> result = new LinkedHashMap<>();
+        if (startMockBackend && !realCredentialsProvided) {
+            MockBackendUtils.logMockBackendUsed();
+
+            try {
+                //copy docker-compose to tmp location
+                File dockerComposeFile, configFile;
+                try (InputStream inYaml = getClass().getClassLoader().getResourceAsStream("servicebus-docker-compose.yaml");
+                        InputStream inJson = getClass().getClassLoader().getResourceAsStream("servicebus-config.json")) {
+                    dockerComposeFile = File.createTempFile("servicebus-docker-compose-", ".yaml");
+                    configFile = File.createTempFile("servicebus-config-", ".json");
+                    Files.copy(inYaml, dockerComposeFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
+                    Files.copy(inJson, configFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
+                }
+
+                container = new ComposeContainer(dockerComposeFile)
+                        .withEnv("ACCEPT_EULA", "Y")
+                        .withEnv("SERVICEBUS_EMULATOR_IMAGE",
+                                config.getValue("servicebus-emulator.container.image", String.class))
+                        .withEnv("SQL_EDGE_IMAGE", config.getValue("azure-sql-edge.container.image", String.class))
+                        .withEnv("CONFIG_FILE", configFile.getAbsolutePath())
+                        .withEnv("MSSQL_SA_PASSWORD", "12345678923456y!43")
+                        .withExposedService("emulator", 5672)
+                        .withLocalCompose(true)
+                        .withLogConsumer("emulator", new Slf4jLogConsumer(LOGGER))
+                        .waitingFor("emulator", Wait.forLogMessage(".*Emulator Service is Successfully Up!.*", 1));
+
+                container.start();
+
+                String connectionString = "Endpoint=sb://%s:%d;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;"
+                        .formatted(container.getServiceHost("emulator", 5672), container.getServicePort("emulator", 5672));
```

Comment:
neat: maybe the port could be a single variable ?

Reply:
`private static int PORT = 5672` if that make sense. Maybe I'm missing something. Plus that's really details, no big deal :)

Reply:
I see what you mean now, I'll do it:)